MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/MotionPhotoInfo.java

```diff
@@ -106,12 +106,12 @@ private static int getVideoOffset(XMPMeta meta) throws XMPException {
                 while (itr.hasNext()) {
                     XMPPropertyInfo property = (XMPPropertyInfo) itr.next();
                     String propertyPath = property.getPath();
-                    String lengthProperty = "Container:Directory["
+                    String lengthProperty = V2_XMP_PROP_PREFIX
                             + arrayItemIdx
-                            + "]/Container:Item/Item:Length";
-                    String paddingProperty = "Container:Directory["
+                            + V2_XMP_PROP_LENGTH_SUFFIX;
+                    String paddingProperty = V2_XMP_PROP_PREFIX
                             + arrayItemIdx
-                            + "]/Container:Item/Item:Padding";
+                            + V2_XMP_PROP_PADDING_SUFFIX;
```

Comment:
move these constants to MotionPhotoInfo since they look specific to this class only and aren't used outside thie class.